### DISCUSSION TITLE
Skip Roadmap?

### DISCUSSION THREAD

**Discussion by haIIux** [skiptools/skip#38] [General]
I'm curious to know if Skip has a roadmap in place or iteration intentions so one may know what Apple based technologies are currently being migrated(?) to be able to be transpiled.

For instance, I will be working with a client shortly on a social media based application for their e-commerce business with Shopify integration. Most of that is fairly straight forward however I have another application which is dependent on Apple Maps and MapKit. It would be awesome to be able to plan my development accordingly in an effort to include that into my project.

**Comment by aabewhite** [comment]
Thanks, that's a good suggestion.

I can tell you that right now we're focused on filling in more of the base Foundation and SwiftUI APIs. We probably won't get to stuff like MapKit for a while. Maybe some member of the community will, though?

Note that as with everything in Skip, it is always possible to use MapKit for your iOS version and use Android's map APIs for your Android version, and then share the rest of the app code. I don't know if that makes sense for your app, but it is almost certainly better than coding the entire app twice.
https://skip.tools/docs/platformcustomization/
